docs/field-details.asciidoc

```diff
@@ -2932,6 +2966,22 @@ example: `Quebec`
 
 // ===============================================================
 
+|
+[[field-geo-timezone]]
+<<field-geo-timezone, geo.timezone>>
+
+| The time zone of the location.
```

Comment:
Worth referencing IANA's timezone db?

Otherwise, PR LGTM.

Reply:
Updated the description with a mention of the IANA time zone names.

I prefer to avoid limiting the field's value exclusively to IANA time zone names, but I know the IANA names will be the most common values here (by a huge margin 😄 ).